internal/config/config.go

```diff
@@ -48,37 +48,6 @@ var (
 	toLogLevel = toLogLevelFromString
 )
 
-// ------------------------------------------------------------------------------------------------
-// Custom GETs
-// ------------------------------------------------------------------------------------------------
```

Comment:
The documentation at the start of this file should be updated now that custom get functions are defined in the `custom_get.go` file.